tests/e2e/utils/helpers.go

```diff
@@ -79,6 +79,14 @@ func GetAPMDir() string {
 	return path.Join(usr.HomeDir, constants.APMDir)
 }
 
+func GetSnapshotsDir() string {
```

Comment:
Seems that this is not part of this PR?

Reply:
yea i accidentally checked in the commits i cherry-picked. reverted.